README.md

```diff
@@ -1,2 +1,150 @@
+    "active": false
+}
+```
+
+A policy objects contains the following fields:
+
+- **uuid** is the unique identifier of the policy. This is automatically
+  generated when a policy is created.
+- **ctime** is the creation timestamp.
+- **name** is a descriptive label associated with the policy. This can be
+  specified when creating a policy. If not specified, the default name
+  `"default"` is used. Names do not need to be unique -- multiple policies can
+  have the same name. This can be used to group policies that are different
+  version of the same logical policy.
+- **type** indicates the policy engine that will be used to evaluate the
+  policy, and therefore the syntax of the policy rules. Currently, the only
+  supported type is `"opa"`.
+- **rules** are the policy rules that will be applied to the attestation
+  result. The format of the rules depends on the policy type. See the Veraison
+  services [OPA policy documentation] for details on how to write "opa"
+  policy rules. You can also seen an example in
+  `misc/example-PSA_IOT-policy.rego`.
+- **active** is a boolean value indicating whether this policy is the current
+  active policy for the scheme. At most one policy is active, however it is
+  also possible for a scheme to have no active policies.
+
+[OPA policy documentation]: https://github.com/veraison/services/blob/main/policy/README.opa.md
+
+
+## Configuration
+
+Configuration for `pocli` is specified using YAML markup language. `polci`
+reads configuration from `config.yaml` in the current working directory and/or
+the user configuration directory (`$XDG_CONFIG_HOME/pocli/config.yaml` on
+Linux). An alternate file may also be specified on the command line with
+`-c`/`--config` option.
+
+The following configuration options are supported:
+
+- **host**: the host name of the Veraison management service. This can also be
+  specified on the command line using `-H`/`--host` option.
+- **port**: the port on which the Veraison management service is listening.
+  This can also be specified on the command line using `-p`/`--port` option.
+
+See `misc/example-config.yaml` for an example configuration file.
+
+
+## Usage examples
+
+All policy management operations are performed with respect to a specific
+attestation scheme. Thus most `pocli` commands take the scheme name as their
+first argument. To see a list of valid scheme names for the service, you can
+execute
+
+    ./pocli schemes
+
+A new policy can be created by passing the name of the attestation scheme to
+which it is to be added and the path to the file containing the policy rules to
+the `create` subcommand:
+
+    ./pocli create PSA_IOT misc/example-PSA_IOT-policy.rego
+
+This will also automatically activate the new policy (which can be suppressed
+with `-d`/`--dont-activate` option). Optionally, the policy can be given a name
+with `-n`/`--name`. If successful, `pocli` will output a JSON dump of the newly
+created policy.
```

Comment:
is the JSON dump on the console or to a supplied file name as argument? kindly clarify

Reply:
There is no option to specify an output file as arguement so it can only be to stdout.